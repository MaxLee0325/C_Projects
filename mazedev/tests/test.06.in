Alpha
Bravo
Charlie
Delta
END
Alpha Bravo
Bravo Charlie
Charlie Delta
Delta Alpha
END
Charlie
Delta
Alpha
Bravo
Charlie
Delta
END
