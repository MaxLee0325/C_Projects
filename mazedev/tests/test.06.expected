Entering room Charlie.
Entering room Delta.
Entering room Alpha.
Entering room Bravo.
Entering room Charlie.
Entering room Delta.
Ended in room Delta.
